src/common/meta/src/ddl/drop_database/executor.rs

```diff
@@ -44,13 +47,15 @@ impl DropDatabaseExecutor {
     /// Returns a new [DropDatabaseExecutor].
     pub fn new(
         table_id: TableId,
+        physical_table_id: TableId,
```

Comment:
What's the difference between the two args `physical_table_id` and `table_id`?

Reply:
The `table_id` could be the `logical_table_id` or `physical_table_id`. The `physical_table_id` converts the `region routes` into the `LogicalTableRouteValue.`